<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>320</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #181818; 
    border: none;
}

QMenuBar {
    background-color: #2c2c2c; 
    color: white;
    font-size: 14px;
    border: none;
}

QMenuBar::item {
    padding: 5px 10px;
}

QMenuBar::item:selected {
    background-color: #444444; 
}

QToolBar {
    background-color: #2c2c2c;
    border: none;
}

QToolButton {
    background-color: #444444; 
    border: none;
    border-radius: 5px;
    color: white;
    font-size: 14px;
    padding: 6px 12px;
}

QToolButton:hover {
    background-color: #555555;
}

QPushButton {
    background-color: #444444;
    border: none;
    border-radius: 5px;
    color: white;
    font-size: 14px;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #555555;
}

QPushButton:pressed {
    background-color: #333333;
}

QLineEdit, QTextEdit, QComboBox {
    background-color: #333333;
    border: 1px solid #555555;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
    color: white;
}

QLineEdit:focus, QTextEdit:focus, QComboBox:focus {
    border-color: #777777;
}

QComboBox::drop-down {
    width: 0px;
    border: none;
    background: none;
}

QComboBox::down-arrow {
    width: 0px;
    height: 0px;
    border: none;
}

QScrollBar:vertical {
    background-color: #333333;
    width: 8px;
    border-radius: 4px;
}

QScrollBar::handle:vertical {
    background-color: #555555;
    border-radius: 4px;
}

QScrollBar::handle:vertical:hover {
    background-color: #666666;
}

QStatusBar {
    background-color: #444444;
    color: white;
    border-top: 1px solid #555555;
}

QStatusBar QLabel {
    color: white;
}

QProgressBar {
    border: 1px solid #555555;
    background-color: #333333;
    border-radius: 5px;
    text-align: center;
    height: 20px;
    color: white;
}

QProgressBar::chunk {
    background-color: #555555;
    border-radius: 5px;
}

QProgressBar::chunk:indeterminate {
    background-color: #666666;
}

QLabel {
    color: white;
    font-size: 14px;
    padding: 5px;
}

QLabel[alignment=&quot;Qt::AlignCenter&quot;] {
    text-align: center;
}

QLabel:hover {
    color: #777777;
}

QLabel[disabled=&quot;true&quot;] {
    color: #888888;
}


QFrame[orientation=&quot;Qt::Horizontal&quot;] {
    height: 2px;
    width: 100%;
}

QFrame[orientation=&quot;Qt::Vertical&quot;] {
    width: 2px;
    height: 100%;
}
</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1,0">
      <item>
       <layout class="QHBoxLayout" name="searchLayout" stretch="0,1,3,1,2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetMinAndMaxSize</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>304</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#qtube-logo {
    font-family: Arial, sans-serif;
    font-size: 36px;
    color: #2c3e50;
	margin-bottom: 0;
	padding-bottom: 0;
}

#qtube-logo span:first-letter {
    font-size: 48px;
    color: #27ae60; 
    font-weight: bold; 
}
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a name=&quot;qtube-logo&quot;/&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#00aa00;&quot;&gt;Q&lt;/span&gt;&lt;span style=&quot; font-size:24pt; font-weight:700;&quot;&gt;Tube&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::RichText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="e_inputUrl">
          <property name="minimumSize">
           <size>
            <width>304</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
          </property>
          <property name="placeholderText">
           <string>Type something for search...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>4</width>
            <height>3</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>627</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>180</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>16</width>
            <height>9</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#imageLabel {
    background-color: #333333;
    border: 0px solid #555555; 
    border-radius: 10px;
    padding: 10px;
    color: white;
}

QLabel#imageLabel:hover {
    border-color: #777777; 
}

QLabel#imageLabel[disabled=&quot;true&quot;] {
    background-color: #222222; 
    border-color: #444444; 
    color: #888888; 
}
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="downloadLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="videoTitleLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>-1</pointsize>
               </font>
              </property>
              <property name="focusPolicy">
               <enum>Qt::FocusPolicy::WheelFocus</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::NoFrame</enum>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Waiting for Video...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chNameLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Waiting for author...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
            <item>
             <widget class="QLabel" name="chViewsLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chDateLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QComboBox" name="comboBoxFormats">
                <property name="editable">
                 <bool>false</bool>
                </property>
                <property name="currentText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="downloadButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Download</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line">
          <property name="styleSheet">
           <string notr="true">QFrame {
    background-color: #555555;
    border: none;
    height: 2px;
    width: 100%;
}
</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
